tests/tests_unit.py

```diff
@@ -128,8 +129,8 @@ def test_custom_variant_array(self):
         self.assertEqual(v, v2)
 
     def test_guid(self):
-        g = ua.Guid()
-        sc = ua.StatusCode()
+        # FIXME what tests to add here?
+        pass
```

Comment:
Well, test what you think is necessary. Create a guid object, convert to binary and back and test it is the same value. For example